src/neo/SmartContract/Native/ManagementContract.cs

```diff
@@ -116,6 +166,8 @@ private ContractState Deploy(ApplicationEngine engine, byte[] nefFile, byte[] ma
             if (md != null)
                 engine.CallFromNativeContract(Hash, hash, md.Name, false);
 
+            engine.SendNotification(Hash, "Deploy", new VM.Types.Array { contract.Hash.ToArray() });
```

Comment:
You must add `CallFlags.AllowNotify` to the method.

Reply:
Done